ui/pages/remote-mode/setup/setup-swaps/index.scss

```diff
@@ -0,0 +1,13 @@
+@use "design-system";
```

Comment:
Is this file being used somewhere? I don't see any imports, so not sure if this gets integrated automatically or what happens.

Reply:
Yeah this automatically gets applied (note the [reference](https://github.com/MetaMask/metamask-extension/pull/32012/files/2574113059fc9d70fb63fafec1aa2e132b429e89#diff-4d0326473b77735227feaf487891a7956994ccf6956e779a66c3cc09916d7830) added to `ui/pages/pages.scss` too)